pkg/virt-config/virt-config.go

```diff
@@ -77,8 +77,8 @@ const (
 	DefaultVirtOperatorLogVerbosity                 = 2
 
 	// Default REST configuration settings
-	DefaultVirtHandlerQPS         float32 = 5
-	DefaultVirtHandlerBurst               = 10
+	DefaultVirtHandlerQPS         float32 = 200
```

Comment:
While the controller and api is cluster wide components, the handler is per node. Do we really see that such high values are needed for the handler? Per node this can go way above. I would like to think that this value needs to be lower than the cluster wide components.

Reply:
I would still half the handler limits, I asked around and I couldn't see any specific testing. Some time ago kubelet had 50/100 limits as default (raised) and I would say we should be close to it.